packages/cli/src/cmds/prune/prune.ts

```diff
@@ -46,31 +46,60 @@ function parseSize(size: string) {
 }
 
 export default {
-  command: 'prune <file> <size>',
+  command: 'prune <file>',
 
-  describe: 'Prune an appmap file down to the given size',
+  describe: 'Make an appmap file smaller by removing events',
 
   builder: (argv: Yargs.Argv) => {
     argv.option('output-dir', {
```

Comment:
There is a standard `--directory, -d` option that we use for the working directory. Please take a look at some of the other commands and you'll see it. Then you can process that option with `handleWorkingDirectory` which will chdir to that directory. It makes it easy, and consistent, to test all these commands against an external repo.